homeassistant/components/discovergy/config_flow.py

```diff
@@ -57,35 +52,26 @@ class DiscovergyConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    _existing_entry: ConfigEntry
-
     async def async_step_user(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
         """Handle the initial step."""
-        if user_input is None:
-            return self.async_show_form(
-                step_id="user",
-                data_schema=CONFIG_SCHEMA,
-            )
-
         return await self._validate_and_save(user_input)
 
     async def async_step_reauth(
         self, entry_data: Mapping[str, Any]
     ) -> ConfigFlowResult:
         """Handle the initial step."""
-        self._existing_entry = self._get_reauth_entry()
         return await self.async_step_reauth_confirm()
 
     async def async_step_reauth_confirm(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
         """Handle the reauth step."""
-        return await self._validate_and_save(user_input, step_id="reauth_confirm")
+        return await self._validate_and_save(user_input)
```

Comment:
The `reauth_confirm` step and `reauth_confirm` strings are now pointless.
You may as well jump to 'user' step directly with `user_input = None`

Reply:
I should also have committed the `strings.json` 🤦